package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "resume-builder",
+  "version": "1.0.0",
+  "description": "Build resumes on the fly",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start",
+    "format": "prettier --write . '**/*.{ts,js,md,yaml,json}'",
+    "format:check": "prettier --check . '**/*.{ts,js,md,yaml,json}'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/leapfrogtechnology/resume-builder"
+  },
+  "author": "Leapfrog Technology",
+  "contributors": [
+    "Yankee Maharjan",
+    "Nikita Shrestha"
+  ],
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/leapfrogtechnology/resume-builder/issues"
+  },
+  "homepage": "https://github.com/leapfrogtechnology/resume-builder#readme",
+  "dependencies": {
+    "eslint-config-leapfrog": "^3.0.0",
+    "next": "^9.3.5",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
+  },
+  "devDependencies": {
```

Comment:
Add husky and lint-staged as well. Refer to the following package.json from reactjs-starter repo. This is still under construction though. eg Prettier is not placed in dev-dependencies. Also there are lots of other  packages which we won;t be using. Be sure to take care of it hai. 

> {
>   "name": "react-starter",
>   "version": "0.1.0",
>   "private": true,
>   "dependencies": {
>     "@emotion/core": "^10.0.28",
>     "@emotion/styled": "^10.0.27",
>     "axios": "^0.19.2",
>     "classnames": "^2.2.6",
>     "formik": "^2.1.4",
>     "husky": "^4.2.5",
>     "lint-staged": "^10.1.3",
>     "moment": "^2.24.0",
>     "node-sass": "^4.13.1",
>     "pinterpolate": "^0.2.1",
>     "prettier": "^2.0.4",
>     "react": "^16.13.1",
>     "react-datepicker": "^2.14.1",
>     "react-dom": "^16.13.1",
>     "react-redux": "^7.2.0",
>     "react-router-dom": "^5.1.2",
>     "react-scripts": "^3.4.1",
>     "react-table": "^7.0.4",
>     "redux": "^4.0.5",
>     "redux-persist": "^6.0.0",
>     "redux-thunk": "^2.3.0",
>     "yup": "^0.28.3"
>   },
>   "lint-staged": {
>     "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
>       "prettier --write",
>       "git add"
>     ]
>   },
>   "scripts": {
>     "start": "react-scripts start",
>     "build": "react-scripts build",
>     "test": "react-scripts test",
>     "eject": "react-scripts eject"
>   },
>   "husky": {
>     "hooks": {
>       "pre-commit": "lint-staged"
>     }
>   },
>   "eslintConfig": {
>     "extends": [
>       "eslint-config-leapfrog/react"
>     ]
>   },
>   "browserslist": {
>     "production": [
>       ">0.2%",
>       "not dead",
>       "not op_mini all"
>     ],
>     "development": [
>       "last 1 chrome version",
>       "last 1 firefox version",
>       "last 1 safari version"
>     ]
>   },
>   "devDependencies": {}
> }
>